core/models.py

```diff
@@ -997,7 +1033,17 @@ class CaseStudy(ClusterableModel):
             heading='Case Study content',
         ),
         MultiFieldPanel(
-            [FieldPanel('hs_code_tags'), FieldPanel('country_code_tags')], heading='Case Study tags for Personalisation'
+            [
+                FieldPanel('hs_code_tags'),
+                FieldPanel('country_code_tags'),
+            ],
+            heading='Case Study tags for Personalisation',
+        ),
+        MultiFieldPanel(
+            [
+                InlinePanel('related_pages', label='Related pages'),
+            ],
+            heading='Related Lesson, Topic & Module',
```

Comment:
Minor: just to make it super-clear to editors

```suggestion
            heading='Related Lesson, Topic & Module, also used for Personalisation',
```